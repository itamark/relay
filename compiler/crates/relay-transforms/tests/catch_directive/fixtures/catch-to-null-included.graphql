fragment EnabledFoo on User {
  name @catch(to: NULL)
}
