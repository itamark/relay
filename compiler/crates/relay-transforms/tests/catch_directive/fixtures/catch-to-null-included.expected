==================================== INPUT ====================================
fragment EnabledFoo on User {
  name @catch(to: NULL)
}
==================================== OUTPUT ===================================
fragment EnabledFoo on User @__childrenCanBubbleNull {
  name @catch(to: NULL) @__CatchMetadataDirective
  # CatchMetadataDirective {
  #     to: Null,
  #     path: "name",
  # }

}
